src/plugins/vis_augmenter/server/saved_objects/augment_vis.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const augmentVisSavedObjectType: SavedObjectsType = {
+  name: 'augment-vis',
+  hidden: false,
+  namespaceType: 'single',
+  management: {
+    // utilizing defaults for some of these fields
+    // icon: 'visualizeApp',
+    // defaultSearchField: 'id',
+    importableAndExportable: true,
+    getTitle(obj) {
+      return 'Augment-' + obj.attributes.visName;
```

Comment:
nit - prefer template literals over string concatenation

```suggestion
      return `Augment-${obj.attributes.visName}`;
```